.gitlab/builds.gitlab-ci.yml

```diff
@@ -12,8 +12,9 @@
   script:
     - git clone https://github.com/flux-framework/flux-core
     - cd flux-core
-    - export FLUX_BUILD_DIR=$(pwd)
     - ./autogen.sh
-    - ./configure
-    - make -j 32
-    - cd ..
+    - mkdir build
+    - cd build
+    - ../configure --prefix=$(pwd)
+    - make -j install
```

Comment:
I would avoid `make -j` since from [make(1)](https://linux.die.net/man/1/make):

>  If the  -j  option  is given without an argument, make will not limit the number of jobs that can run simultaneously.

Maybe `make -j $(nproc)` would be better.

Does this part of the build take place on a login node, or in the allocated nodes for the CI run?

Reply:
Everything always happens in allocated debug nodes in our CI. But I'll change to `make -j $(nproc)`, that does what I thought I was doing here.